xds/internal/xdsclient/clientimpl_watchers.go

```diff
@@ -48,17 +48,6 @@ func (c *clientImpl) WatchResource(rType xdsresource.Type, resourceName string,
 		return func() {}
 	}
 
-	// TODO: replace this with the code does the following when we have
-	// implemented generic watch API on the authority:
-	//  - Parse the resource name and extract the authority.
-	//  - Locate the corresponding authority object and acquire a reference to
-	//    it. If the authority is not found, error out.
-	//  - Call the watchResource() method on the authority.
-	//  - Return a cancel function to cancel the watch on the authority and to
-	//    release the reference.
-
-	// TODO: Make ParseName return an error if parsing fails, and
-	// schedule the OnError callback in that case.
```

Comment:
Is this done (via the `findAuthority` check)?

Reply:
Oh let me re add this comment Easwar wrote a resource not found trigger that didn't work in my use case which I copied than changed back which involved deleting this comment.